submissions/nhunzaker/lib/server.js

```diff
@@ -0,0 +1,28 @@
+var Hapi = require('hapi')
+var pkg = require('../package')
+
+exports.metadata = {
```

Comment:
Is `exports.metadata` and `exports.init` and convention you picked up from other `Hapi` projects?